Description: Update the spooldir from /usr/spool to /var/spool.
Forwarded: not-needed

---
 alpine/pine-use.c             |    2 +-
 doc/man1/alpine.1             |    2 +-
 doc/tech-notes/config.html    |    2 +-
 doc/tech-notes/tech-notes.txt |    2 +-
 imap/docs/imaprc.txt          |   10 +++++-----
 imap/docs/locking.txt         |    6 +++---
 imap/src/osdep/unix/Makefile  |    6 +++---
 pith/conf.c                   |    4 ++--
 8 files changed, 17 insertions(+), 17 deletions(-)

--- a/imap/docs/imaprc.txt	2022-06-09 01:10:39.303756744 -0400
+++ b/imap/docs/imaprc.txt	2022-06-09 01:10:39.243756955 -0400
@@ -168,8 +168,8 @@
    Sets the directory in which the user's data can be found.  A user's
    folders can be found in a subdirectory of the black box directory
    named with the user's username.  For example, if the blackbox
-   directory is /usr/spool/folders/, user jones' data can be found
-   in /usr/spool/folders/jones/.  The user's black-box directory is
+   directory is /var/spool/folders/, user jones' data can be found
+   in /var/spool/folders/jones/.  The user's black-box directory is
    the location of folders, .mminit, .imaprc, .newsrc, and all other
    files used by c-client; internally, it sets c-client's idea of the
    user's ``home directory'', overriding /etc/passwd.
@@ -411,14 +411,14 @@
     If non-zero, a warning message is given if an attempt to create a
     lock file fails.  Otherwise, EACCES is treated as a "silent failure",
     and it proceeds without trying to use the lock file.  This is for
-    the benefit of users on systems with paranoid /usr/spool/mail
-    protections which don't let users create /usr/spool/mail/$(USER).lock
+    the benefit of users on systems with paranoid /var/spool/mail
+    protections which don't let users create /var/spool/mail/$(USER).lock
     files; these unfortunate users would be harassed with a flood of
     error messages otherwise.  The problem is that on SVR4, if EACCES
     remains disabled and fcntl() locking is also disabled, then there is
     no locking at all which is doubleplus-ungood.
 
-    If the site is paranoid on /usr/spool/mail protections AND if there
+    If the site is paranoid on /var/spool/mail protections AND if there
     is no fcntl() locking (SVR4) or usable flock() locking (e.g. NFS),
     then there is no way to win.  Find a different system to use.
 
--- a/imap/docs/locking.txt	2022-06-09 01:10:39.303756744 -0400
+++ b/imap/docs/locking.txt	2022-06-09 01:10:39.287756801 -0400
@@ -255,7 +255,7 @@
 timeout for this lock, after which it is broken on the presumption
 that it is a stale lock.  If it can not create the .lock file due to
 an EACCES (protection failure) error, it once silently proceeded
-without this lock; this was for systems which protect /usr/spool/mail
+without this lock; this was for systems which protect /var/spool/mail
 from unprivileged processes creating files.  Today, c-client reports
 an error unless it is built otherwise.  The purpose of this lock is to
 prevent against unfavorable interactions with mail delivery.
@@ -278,7 +278,7 @@
 
      Mail delivery daemons use lock (1), (2), or both.  Lock (1) works
 over NFS; lock (2) is the only one that works on sites that protect
-/usr/spool/mail against unprivileged file creation.  Prudent mail
+/var/spool/mail against unprivileged file creation.  Prudent mail
 delivery daemons use both forms of locking, and of course so does
 c-client.
 
@@ -335,7 +335,7 @@
 the mail file is NFS-mounted.
 
      What this means is that there is *no* locking protection at all
-in the case of a client using an NFS-mounted /usr/spool/mail that does
+in the case of a client using an NFS-mounted /var/spool/mail that does
 not permit file creation by unprivileged programs.  It is impossible,
 under these circumstances, for an unprivileged program to do anything
 about it.  Worse, if EACCES errors on .lock file creation are no-op'ed
--- a/alpine/pine-use.c	2022-06-09 01:10:39.303756744 -0400
+++ b/alpine/pine-use.c	2022-06-09 01:10:39.287756801 -0400
@@ -20,7 +20,7 @@
 #include <sys/stat.h>
 
 #ifndef MAILSPOOLPCTS
-#define MAILSPOOLPCTS "/usr/spool/mail/%s"
+#define MAILSPOOLPCTS "/var/spool/mail/%s"
 /* #define MAILSPOOLPCTS "/usr/mail/%s" */
 #endif
 
--- a/doc/man1/alpine.1	2022-06-09 01:10:39.303756744 -0400
+++ b/doc/man1/alpine.1	2022-06-09 01:10:39.287756801 -0400
@@ -318,7 +318,7 @@
 .if n .ta 2.8i
 .if t .ta 2.1i
 
-/usr/spool/mail/xxxx	Default folder for incoming mail.
+/var/spool/mail/xxxx	Default folder for incoming mail.
 .br
 ~/mail	Default directory for mail folders.
 .br
--- a/doc/tech-notes/config.html	2022-06-09 01:10:39.303756744 -0400
+++ b/doc/tech-notes/config.html	2022-06-09 01:10:39.291756786 -0400
@@ -2326,7 +2326,7 @@
 
 <DD> This option tells <EM>Alpine</EM> where to look for the "news spool" for
 newsgroups when accessing news locally, rather than via NNTP.  The default
-path is usually <CODE>/usr/spool/news</CODE>. <P>
+path is usually <CODE>/var/spool/news</CODE>. <P>
 
 <DT> <A NAME="newsrc-path"><EM>newsrc-path</EM></A>
 
--- a/doc/tech-notes/tech-notes.txt	2022-06-09 01:10:39.303756744 -0400
+++ b/doc/tech-notes/tech-notes.txt	2022-06-09 01:10:39.295756772 -0400
@@ -2905,7 +2905,7 @@
    _news-spool-directory_
           This option tells _Alpine_ where to look for the "news spool"
           for newsgroups when accessing news locally, rather than via
-          NNTP. The default path is usually /usr/spool/news.
+          NNTP. The default path is usually /var/spool/news.
    _newsrc-path_
           This option overrides the default name _Alpine_ uses for your
           "newsrc" news status and subscription file. If set, _Alpine_
--- a/imap/src/osdep/unix/Makefile	2022-06-09 01:10:39.303756744 -0400
+++ b/imap/src/osdep/unix/Makefile	2022-06-09 01:10:39.295756772 -0400
@@ -99,7 +99,7 @@
 SIGTYPE=bsd
 CRXTYPE=std
 ACTIVEFILE=/usr/lib/news/active
-SPOOLDIR=/usr/spool
+SPOOLDIR=/var/spool
 MAILSPOOL=$(SPOOLDIR)/mail
 NEWSSPOOL=$(SPOOLDIR)/news
 RSHPATH=/usr/ucb/rsh
@@ -234,7 +234,7 @@
 asv:	# Altos SVR4
 	$(BUILD) `$(CAT) SPECIALS` OS=$@ \
 	 SIGTYPE=sv4 LOGINPW=old \
-	 ACTIVEFILE=/usr/spool/news/active \
+	 ACTIVEFILE=/var/spool/news/active \
 	 RSHPATH=/usr/bin/rcmd \
 	 BASECFLAGS="-Dconst= -DSIGSTOP=SIGKILL" \
 	 BASELDFLAGS="-lsocket -lrpc -lgen -lcrypt -lxenix" \
@@ -371,7 +371,7 @@
 gas:	# GCC Altos SVR4
 	$(BUILD) `$(CAT) SPECIALS` OS=asv \
 	 SIGTYPE=sv4 LOGINPW=old \
-	 ACTIVEFILE=/usr/spool/news/active \
+	 ACTIVEFILE=/var/spool/news/active \
 	 RSHPATH=/usr/bin/rcmd \
 	 BASECFLAGS="-g -O -DALTOS_SYSTEM_V -DSIGSTOP=SIGKILL" \
 	 BASELDFLAGS="-lsocket -lrpc -lgen -lcrypt -lxenix" \
--- a/pith/conf.c	2022-06-09 01:10:39.303756744 -0400
+++ b/pith/conf.c	2022-06-09 01:10:39.299756758 -0400
@@ -137,7 +137,7 @@
 
 CONF_TXT_T cf_text_wp_columns[] =		"Web Alpine preferred width for message display in characters";
 
-CONF_TXT_T cf_text_inbox_path[] =		"Path of (local or remote) INBOX, e.g. ={mail.somewhere.edu}inbox\n# Normal Unix default is the local INBOX (usually /usr/spool/mail/$USER).";
+CONF_TXT_T cf_text_inbox_path[] =		"Path of (local or remote) INBOX, e.g. ={mail.somewhere.edu}inbox\n# Normal Unix default is the local INBOX (usually /var/spool/mail/$USER).";
 
 CONF_TXT_T cf_text_incoming_folders[] =	"List of incoming msg folders besides INBOX, e.g. ={host2}inbox, {host3}inbox\n# Syntax: optnl-label {optnl-imap-host-name}folder-path";
 
@@ -357,7 +357,7 @@
 
 CONF_TXT_T cf_text_news_active[] =	"Path and filename of news configuration's active file.\n# The default is typically \"/usr/lib/news/active\".";
 
-CONF_TXT_T cf_text_news_spooldir[] =	"Directory containing system's news data.\n# The default is typically \"/usr/spool/news\"";
+CONF_TXT_T cf_text_news_spooldir[] =	"Directory containing system's news data.\n# The default is typically \"/var/spool/news\"";
 
 CONF_TXT_T cf_text_upload_cmd[] =	"Path and filename of the program used to upload text from your terminal\n# emulator's into Alpine's composer.";
 
